posts/2022-04-15-libertydevops-gitjenkinshelmiks.adoc

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: "Continuous integration and delivery (CI/CD) of Jakarta EE & MicroProfile applications with Open Liberty to Kubernetes by using Jenkins"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/shamjithantholi
+author_github: https://github.com/shamjithantholi
+seo-title: TITLE - OpenLiberty.io
+seo-description: DESCRIPTION
+blog_description: "A high level explanation about how to build a devops pipeline with Jenkins as the core controller component "
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Continuous integration and delivery (CI/CD) of Jakarta EE & MicroProfile applications with Open Liberty to Kubernetes by using Jenkins
+Shamjith Antholi https://github.com/shamjithantholi
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+[#Intro]
+== DevOps, CI/CD, Jenkins & OpenLiberty ( to deploy Jakarta EE & MicroProfile application)
```

Comment:
Remove the first sub-heading. It's not necessary and not part of our house style. Instead we just need a punchy, search engine-friendly opening paragraph.

Reply:
removed the first heading